### PR TITLE
Add pygrib package to requirements file

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Add missing third party package to requirements file

### What the code is doing
N/A

### Testing
N/A

### Where to look
`pygrib` is added to the *requirements.txt*

### Usage Example/Visuals
N/A

### Time estimate
2min

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ black
 pytest
 coverage
 pytest-cov
+pygrib~=2.1.4
```
Comment (BainanXia) [inline requirements.txt:19]: Not familiar with the versions of this package. Is it safe to do `~=` rather than `==`?

Comment (danielolsen) [reply requirements.txt:19]: `~=` on a patch version _shouldn't_ cause any problems. The Pipfile just specifies `pygrib = "*"` so we're being more cautious here already.

Comment (rouille) [reply requirements.txt:19]: Good point. I was leaning on the side of some flexibility but I cannot guarantee we won't run into problems with later version. I am fine with locking the version. Any suggestion @jon-hagg?

Comment (BainanXia) [reply requirements.txt:19]: @danielolsen I checked `Pipfile.lock` which chooses 2.1.4 and brought up the conversation. Maybe it doesn't matter.

**Review by danielolsen — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (jenhagg) [reply requirements.txt:19]: I think `~=` is a good middle ground - safer than `*` and less work to maintain than `==`. That's what i'd use by default, and only use `==` for special cases or unstable packages.

**Review by BainanXia — APPROVED** [review]

**Pull request merged**